plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -0,0 +1,76 @@
+import ast
+import re
+from operator import itemgetter
+
+import insightconnect_plugin_runtime
+from typing import Dict, List
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+from insightconnect_plugin_runtime.helper import get_time_hours_ago
+
+from .schema import MonitorSiemLogsInput, MonitorSiemLogsOutput, MonitorSiemLogsState, Component
+from ...util.event import EventLogs
+from ...util.exceptions import ApiClientException
+
+
+class MonitorSiemLogs(insightconnect_plugin_runtime.Task):
+    NEXT_TOKEN = "next_token"  # nosec
+    HEADER_NEXT_TOKEN = "mc-siem-token"  # nosec
+    STATUS_CODE = "status_code"  # nosec
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="monitor_siem_logs",
+            description=Component.DESCRIPTION,
+            input=MonitorSiemLogsInput(),
+            output=MonitorSiemLogsOutput(),
+            state=MonitorSiemLogsState(),
+        )
+
+    def run(self, params={}, state={}) -> (List[Dict], Dict):
+        if not state:
+            self.logger.info("First run")
+        else:
+            self.logger.info("Subsequent run")
+            params["token"] = state.get(self.NEXT_TOKEN)
```

Comment:
It may be good to use a constant here for the `"token"` value as it's re-used on line 46